Riot/Modules/Pills/PillsFormatter.swift

```diff
@@ -176,3 +224,34 @@ extension PillsFormatter {
         return string
     }
 }
+
+@available(iOS 15.0, *)
+private extension PillsFormatter {
+    static func markdownUrls(in attributedString: NSAttributedString) -> [(url: URL, label: String, range: NSRange)] {
+        // Create a regexp that detects markdown links.
+        let pattern = "\\[([^\\]]+)\\]\\(([^\\)\"\\s]+)(?:\\s+\"(.*)\")?\\)"
```

Comment:
Swift support [regex literals](https://www.hackingwithswift.com/swift/5.7/regexes) now, can we use that to improve readability?

Reply:
Actually we discussed this great new addition to Swift at one of our iOS weekly syncs, will be great to transition to this for Element-X, but unfortunately all these Regex and RegexBuilder objects are only available with iOS 16.0+